app/assets/src/helpers/ObjectHelper.js

```diff
@@ -26,6 +27,10 @@ class ObjectHelper {
   static deepCopy(obj) {
     return JSON.parse(JSON.stringify(obj));
   }
+  // Sets the value at path of object, and returns new object.
```

Comment:
Why do need this? You do not like original signature?

Reply:
i guess this is about that conversation on lodash/fp... i see

Reply:
I would skip it... might be confusing, but has documentation, no?
I just think that the lack of consistency - some times we encapsulate other we use the functions from lodash directly - will make it more confusing.